code/modules/clothing/head/helmet.dm

```diff
@@ -718,6 +713,7 @@ GLOBAL_LIST_INIT(allowed_helmet_items, list(
 	icon_state = "tech_helmet"
 	specialty = "M10 technician"
 	built_in_visors = list(new /obj/item/device/helmet_visor, new /obj/item/device/helmet_visor/welding_visor)
+	built_in_visors = list(new /obj/item/device/helmet_visor/welding_visor)
```

Comment:
?

Reply:
must have accidentally copied that instead of removing helmet visor, oops.